app/components/widgets/forms/image-upload.js

```diff
@@ -8,6 +8,7 @@ export default Component.extend({
   selectedImage   : null,
   allowDragDrop   : true,
   requiresDivider : false,
+  aspectRatio     : [2, 1],
```

Comment:
What is this change and where is the square cropper now?

Reply:
this is for create event page, . I have made it square for speaker. 2:1 is for - 
![Screenshot from 2020-01-21 20-12-57](https://user-images.githubusercontent.com/56407566/72814071-72f47c00-3c8a-11ea-9471-4b03aa3f82a8.png)

Reply:
This is unneeded. This should be the default value of prop, and should not be needed to write explicitly